pubsub/pubsub.go

```diff
@@ -532,6 +534,20 @@ func (s *Subscription) Receive(ctx context.Context) (_ *Message, err error) {
 			m := s.q[0]
 			s.q = s.q[1:]
 			s.throughputCount++
+			if s.ackFunc == nil {
+				// Add a finalizer that complains if the Message we return isn't
+				// acked or nacked.
+				_, file, lineno, ok := runtime.Caller(1)
```

Comment:
I assume this refers to Receive's caller? Can you document that?

Reply:
Yes, done.